drift/app.py

```diff
@@ -22,8 +21,9 @@ def create_app():
     """
     app_name = os.getenv("APP_NAME", "drift")
     openapi_args = {"path_prefix": config.path_prefix, "app_name": app_name}
-    connexion_app = connexion.App(__name__, specification_dir="openapi/", arguments=openapi_args)
-    connexion_app.add_api("api.spec.yaml", validate_responses=True, strict_validation=True)
+    connexion_app = FlaskApp(__name__, specification_dir="openapi/", arguments=openapi_args)
+    # connexion_app.add_api("api.spec.yaml", validate_responses=True, strict_validation=True)
```

Comment:
I'd recommend to remove code in the comment and rather mention that response validation is turned off.